superbench/benchmarks/micro_benchmarks/dist_inference.py

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
Do we need to revise document?

Reply:
Add something in doc to describe new cpp implementation